code/datums/objective/individual_objective/technomancers.dm

```diff
@@ -8,7 +8,7 @@
 
 /datum/individual_objective/disturbance/assign()
 	..()
-	target_area = random_ship_area()
+	target_area = random_ship_area(FALSE, TRUE, FALSE, TRUE)
```

Comment:
```suggestion
	target_area = random_ship_area(need_apc=TRUE)
```
Since we are keeping the same values for the other default arguments there is no need to put them again.